services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,19 +43,52 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
```

Comment:
check to make sure that the queue has been created so `thermalMgrQueueHandle` is not NULL